### PR TITLE
vulnerable dependencies upgrade/ fixes #114

### LINKED ISSUE #114: js: github complains about vulnerable jquery, upgrade to > 3.0
jQuery 2.x is not maintained any more, update the package.json, test, ...

### PR DESCRIPTION
**Pull request by Saman-jafari**

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -24,17 +24,18 @@
   },
   "homepage": "https://github.com/borgbackup/borgweb#readme",
   "devDependencies": {
-    "babelify": "^6.1.2",
-    "browser-sync": "^2.7.13",
-    "browserify": "^10.2.4",
+    "@babel/core": "^7.0.0-beta.51",
```
Comment (ThomasWaldmann) [inline js/package.json]: this is a new dependency and it is beta. is listing it necessary and could we use something non-beta?

File: js/package.json [hunk 2]
```diff
@@ -24,17 +24,18 @@
...
-    "gulp": "^3.9.0",
+    "gulp": "^4.0.0",
     "gulp-util": "^3.0.5",
     "minifyify": "^7.0.1",
     "prependify": "^1.1.4"
   },
   "dependencies": {
     "bootstrap": "^3.3.5",
-    "jquery": "^2.1.4"
+    "jquery": "^3.3.1"
```
Comment (ThomasWaldmann) [inline js/package.json]: did you test borgweb whether it works after updating the js?

i guess some js files need regenerating now?

Comment (Saman-jafari) [reply js/package.json]: yes you need to regenerate the bundle but it worked I tested it out everything is working for gulp four some new things changed that I checked in your gulp js and everything was normal so no need to worry about it

File: js/package.json (hunk 1 above)
Comment (Saman-jafari) [reply js/package.json]: babel 7 is in just beta but it works fine if you go to V6 some error rise I used V7 beta for a while now it works fine and also has not any security issue
good thing is package json  had more vulnerability and also high level ones than just jquery now 2 low ones left :)

**Comment by Saman-jafari** [comment]
OK I changed minor things in gulp file to cover IE also now it works like a charm :)

**Comment by ThomasWaldmann** [comment]
could you please put the source file changes into 1 commit and the generated stuff in another?

**Comment by Saman-jafari** [comment]
Done

**Comment by ThomasWaldmann** [comment]
cool, thanks!

**Pull request merged**